### PR TITLE
python3packages.pyotgw: unstable-2021-03-25

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python module to interact the OpenTherm Gateway

https://github.com/mvn23/pyotgw

This is a Home Assistant dependency. Not sure, why `parse-requirements.py` is not able to pick it up. Could be the same issue as with `slackclient`.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 120278` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.pyotgw</li>
    <li>python39Packages.pyotgw</li>
  </ul>
</details>

**Comment by mweinelt** [comment]
> python3.8-pyotgw-unstable-2021-03-25

Pretty sure it is because of the unstable version.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 120278` at 1eb8da2a run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.pyotgw</li>
    <li>python39Packages.pyotgw</li>
  </ul>
</details>

File: pkgs/development/python-modules/pyotgw/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+, fetchFromGitHub
+, pyserial-asyncio
+, pytest-asyncio
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "pyotgw";
+  version = "unstable-2021-03-25";
```
Comment (mweinelt) [inline pkgs/development/python-modules/pyotgw/default.nix:12]: Why not 1.1b1?

**Comment by mweinelt** [comment]
Rebase on top of #120281, which will fix this occasion. I'm still looking into the mentioned slackclient issue.

File: pkgs/development/python-modules/pyotgw/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/pyotgw/default.nix:12]: The tests were added after 1.1b1.

**Comment by mweinelt** [comment]
This needs to be rebased and updated.

File: pkgs/development/python-modules/pyotgw/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/pyotgw/default.nix:12]: Request for new release: https://github.com/mvn23/pyotgw/issues/33

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 120278` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.pyotgw</li>
    <li>python39Packages.pyotgw</li>
  </ul>
</details>

**Comment by mweinelt** [comment]
@GrahamcOfBorg test home-assistant

**Pull request merged**